### PR TITLE
Unlock NFT Feature

### PR DESCRIPTION
**Pull request by gaupoit**
### Summary (Please recap what you changed or fixed)

- NFT Welcome Screen
- NFT tab

### Checklist (If you won't pass this checklist please comment why)

-   [x] I removed all commented or unnecessary code.
-   [x] Provide the screenshots due to UI changed or bug fixed
-   [x] My code has covered these test cases (please list down your tested cases):

Simulator Screen Shot - iPhone 13 - 2022-03-29 at 09 29 18
Simulator Screen Shot - iPhone 13 - 2022-03-29 at 09 22 31

### CODE REVIEW

File: src/screens/authentication/create_new_wallet/AddKeyScreen/index.tsx [hunk 1]
```diff
@@ -23,12 +23,14 @@ function AddKeyScreen() {
   const dispatch = useDispatch();
 
   const onChange = (value?: string) => {
-    setPublicKey(value ?? '');
-    setError(isValidPublicKey(value) ? '' : 'Invalid public key');
+    setPublicKey('02021172744b5e6bdc83a591b75765712e068e5d40a3be8ae360274fb26503b4ad38');
+    // setError(isValidPublicKey(value) ? '' : 'Invalid public key');
```
Comment (djhahe) [inline src/screens/authentication/create_new_wallet/AddKeyScreen/index.tsx:27]: this is for debugging ? pls revert

File: src/screens/authentication/create_new_wallet/AddKeyScreen/index.tsx [hunk 2]
```diff
@@ -23,12 +23,14 @@ function AddKeyScreen() {
...
   };
 
   const onAddPublicKey = async () => {
     setLoading(true);
+    console.log('Hi', publicKey);
```
Comment (djhahe) [inline src/screens/authentication/create_new_wallet/AddKeyScreen/index.tsx:32]: Pls remove

**Review by djhahe — APPROVED** [review]

File: src/screens/authentication/create_new_wallet/AddKeyScreen/index.tsx [hunk 3]
```diff
@@ -29,6 +29,7 @@ function AddKeyScreen() {
 
   const onAddPublicKey = async () => {
     setLoading(true);
+
     dispatch(
```
Comment (gaupoit) [inline src/screens/authentication/create_new_wallet/AddKeyScreen/index.tsx:33]: _test comment_

Comment (gaupoit) [reply src/screens/authentication/create_new_wallet/AddKeyScreen/index.tsx:33]: hi

**Pull request merged**